### DISCUSSION TITLE
Is there a way to have Watchtower wait for an update to be a minimum age before updating?

### DISCUSSION THREAD

**Discussion by davin900** [containrrr/watchtower#1879] [Q&A]
I looked through the documentation and pretty sure I didn't find anything but wanted to ask...

I have recently become a bit concerned about the possibility of updating a container and having some kind of breaking change happen... or some kind of bug in an update.

So I was thinking a nice safeguard might be to set a minimum age for a new version of a container. Like if a new version hasn't been retracted or taken down after... 3 days... then it updates?

Is that a silly thing to fear? I mostly use Docker for things like Plex, Radarr, Sonarr, Calibre, etc.

**Comment by pjdubya** [comment]
I've been working on a change for exactly this because I have the same use case and I saw a few other users discussing it too. It is working well now and I plan to submit a pull request once I figure out how to create branches here. (If anyone reading this can tell me how to get permission to create branches, please let me know.)

**Comment by waja** [comment]
> I plan to submit a pull request once I figure out how to create branches here.

https://docs.github.com/articles/creating-a-pull-request

**Reply by pjdubya** [reply]
@waja  thanks, but what I meant is that I don't have permission to create branches in this repo as I've never contributed here before. When I try to do so via git cli I get a 403. When I try via the github browser interface, there's no "New branch" button for this repo for me.

**Reply by waja** [reply]
You need to create a fork, which is also written in the documentation.

**Comment by wreckedcarzz** [comment]
@pjdubya any progress with your pr? I just had this situation happen this morning and am seeking this functionality.

**Reply by pjdubya** [reply]
Yup, I submitted the PR (https://github.com/containrrr/watchtower/pull/1884) last month. @piksel reviewed that and realized it was better to first refactor a few things and created a PR for that (https://github.com/containrrr/watchtower/pull/1895) but it's still marked draft as of now. I think once piksel's PR gets merged mine should be able to merge very soon after that once I update my PR to match those updates.
